src/main/kotlin/day19/Food Recipe App.md

```
@@ -0,0 +1,27 @@
+# 메인 메뉴 객체
+- 레시피 정보
+
+
+# 검색 객체
+- 레시피 썸네일들
+- # 레시피 프로필 객체
+- # ''' 메뉴 객체
+
+
+# 저장된 레시피 객체
+- # 레시피 프로필 객체
```

Markdownlint (0.37.0)

Multiple headings with the same content

(MD024, no-duplicate-heading)